### PR TITLE
Use a dedicated token for the ontobot workflow.

### PR DESCRIPTION
**Pull request by gouttegd**
This commit updates the ontobot workflow (`auto-pr-ontobot`) to make it use a dedicated authentication token (`ONTOBOT_TOKEN`), rather than the default `GITHUB_TOKEN`.

This should ensures that PRs created by the bot can trigger other workflows, notably the CI workflow.

related to #3223

### CODE REVIEW

File: .github/workflows/auto-pr-ontobot.yml
Comment (hrshdhgd) [inline .github/workflows/auto-pr-ontobot.yml:60]: This may not be needed either.

**Review by hrshdhgd — APPROVED** [review]
The 2 lines that I've written comments on were commented before because I was testing if those authentication lines were needed or no. I think we can get away without using them. The rest looks good.

**Comment by gouttegd** [comment]
Yes, I suspected that the lines you flagged might not be necessary – the standard permissions granted by the default `GITHUB_TOKEN` should be enough for what those steps are doing. Still, I like the consistency of using the same token throughout the workflow.

(There’s a release ongoing, so I’ll wait for the release to be over before merging this.)

**Comment by hrshdhgd** [comment]
Sounds good!

**Review by anitacaron — APPROVED** [review]

**Pull request merged**
